src/interaction/orbitalnavigator.cpp

```diff
@@ -136,8 +135,45 @@ namespace {
         "" // @TODO Missing documentation
     };
 
-    constexpr openspace::properties::Property::PropertyInfo StereoInterpolationTimeInfo =
-    {
+    constexpr openspace::properties::Property::PropertyInfo VelocityZoomControlInfo = {
+        "VelocityZoomControl",
+        "Velocity zoom control",
+        "Controls the velocity of the camera motion when zooming in to the focus node. "
+        "The higher the value the faster the camera will move towards the focus."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo FlyToNodeInfo = {
+        "FlyToNode",
+        "Fly to focus node",
+        "Determines whether to fly to the focus node or just focus on it."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo OverviewInfo = {
+        "Overview",
+        "Overview",
+        "Determines whether to zoom out to an overview or not."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo ApplyOverviewInfo = {
+        "ApplyOverview",
+        "Apply overview ",
+        "Triggering this property makes the camera move to an overview."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo ApplyFlyToInfo = {
+        "ApplyFlyTo",
+        "Apply fly to node",
+        "Triggering this property makes the camera move to the focus node."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo StoryOverviewLimitInfo = {
```

Comment:
It would be great if we can figure out how this can be either moved out to a class that cares about stories, or if the concept can be generalized to be useful for non-story use-cases.